guillotina/component/hooks.py

```diff
@@ -14,13 +14,13 @@
 # flake8: noqa
 __docformat__ = 'restructuredtext'
 
-import contextlib
-import threading
-
-from guillotina.component.globalregistry import getGlobalSiteManager
+from guillotina.component.globalregistry import get_global_components
 from guillotina.component.interfaces import ComponentLookupError
 from guillotina.component.interfaces import IComponentLookup
 
+import contextlib
+import threading
```

Comment:
Can we remove the SetSite and the threading import ?

Reply:
I thought I'd leave it because I was worried about what happens with executors...

Reply:
I can't understand why its involved with executors.

Reply:
What is wrong with thread locals?